### ISSUE TITLE
Specification of chanis in Markov-chain

### ISSUE THREAD

**Issue by narmindavoudi** [jmaih/RISE_toolbox#167]
Dear Maih,
 At firs, thank you for your help about working on rise-toolbox.
I have a question about acceptance rate in Metropolis Hasting algorithm,
1)after estimating and simulating the posterior distributions, in set-upping the options for this algorithm, I want to be able to change the number of chins in Markov chain so that acceptance rate to stand between 0.3 and 0.5 as the same to  dynare ,could you please guide me how I can do that.
actually, results of plot of posteriors are not in the standard shape, and I guess it is because of acceptance rate which can be controlled by changing of chain number.
2) I chose100.000 for "ndraws_mcmc" and I wonder if is there any reasonable number of drawing parameters through MCMC to reach a standard shape of posteriors density?
Best regard
Narmin

**Comment by jmaih** [comment]
Dear Narmin,

If you run "help mh_sampler"  in the command window you should be able to
see the various options you can set in order to change. In your specific
case, the option is called "nchain".

The option for controlling the acceptance rate is called "alpha", which you
should set as an interval. For instance, options.alpha=[.25,.45] . The
sampler will then target the middle of the interval

With respect to results of the posterior plots, they will depend on the
sampler you use, the number of draws you consider and the various options
you set.

There is no standard number of draws one should use. Various people use
different numbers often depending on the time budget at hand and how hard
it is to compute the likelihood.

If you are now doing any thinning I think 100 000 draws is not enough. The
metropolis-hastings algorithm will typically give you highly correlated
draws.

In summary, look up the various options you can set in the help for the
mh_sampler.

Cheers,

J.

**Comment by narmindavoudi** [comment]
Thank you for your help.
best regard
Narmin
